localstripe/resources.py

```diff
@@ -2361,9 +2360,16 @@ def _update(self, metadata=None, items=None, trial_end=None,
         except AssertionError:
             raise UserError(400, 'Bad request')
 
-        if items is None or len(items) != 1 or not items[0]['plan']:
+        if tax_percent is not None:
+            self.tax_percent = tax_percent
```

Comment:
I don't understand this change, why is it?

Reply:
Good catch - believe it was a bad conflict resolution when I merged